k2/csrc/fsa_algo.h

```diff
@@ -478,6 +475,21 @@ Fsa LinearFsa(const Array1<int32_t> &symbols);
  */
 FsaVec LinearFsas(const Ragged<int32_t> &symbols);
 
+/*
+  Create an FsaVec containing ctc graph FSAs, given a list of sequences of
+  symbols
+
+    @param [in] symbols  Input symbol sequences (must not contain
```

Comment:
Do we need to add a check inside the kernel that none of the input symbols is -1?

Reply:
I add a checking in the kernel `set_num_arcs`, which will enumerate all the symbols. I think it's enough.